framework/basehandlers.py

```diff
@@ -573,6 +573,14 @@ def get_template_data(self, **defaults):
           self, feature_id)
       if redirect_resp:
         return redirect_resp
+    # Validate the user has admin permissions and redirect if needed.
+    if defaults.get('require_admin_site'):
+      user = self.get_current_user()
+      # Should have already done the require_signin check.
+      # If for reason, we don't let's treat it as the main 403 case.
+      if not user \
```

Comment:
Style: Instead of using `\` for line continuation, wrap the expression in parens.
https://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
Good catch!